client/my-sites/upgrades/checkout/thank-you.jsx

```diff
@@ -63,6 +66,24 @@ var CheckoutThankYou = React.createClass( {
 	},
 
 	componentDidMount: function() {
+		var lastTransaction = this.props.lastTransaction,
+			selectedSite;
+
+		if ( lastTransaction ) {
```

Comment:
Do we need to guard against a missing `lastTransaction` here? There are a couple other places in this file where we assume that it is present.